tests/unit/data-access/FetchSearchEntityRepository.spec.ts

```diff
@@ -66,6 +66,7 @@ describe( 'FetchSearchEntityRepository', () => {
 			origin: '*',
 			limit,
 			offset,
```

Comment:
The offset here is not needed at all: https://www.wikidata.org/w/api.php?action=help&modules=wbsearchentities

Reply:
good catch